pkg/auth/jwt.go

```diff
@@ -0,0 +1,68 @@
+package auth
+
+import (
+	"fmt"
+	"net/http"
+
+	"NKSS-backend/pkg/handlers"
+
+	"github.com/golang-jwt/jwt"
+	_ "github.com/lib/pq"
+)
+
+func Authenticator(next http.Handler, secret string) http.Handler {
```

Comment:
The next handler needs to know the the roll number. You will need to make `passDB` and `Authenticator` composable somehow.